crates/rbuilder/src/live_builder/order_input/txpool_fetcher.rs

```diff
@@ -24,10 +24,20 @@ pub async fn subscribe_to_txpool_with_blobs(
     results: mpsc::Sender<ReplaceableOrderPoolCommand>,
     global_cancel: CancellationToken,
 ) -> eyre::Result<JoinHandle<()>> {
-    let ipc = IpcConnect::new(config.ipc_path);
-    let provider = ProviderBuilder::new().on_ipc(ipc).await?;
-
     let handle = tokio::spawn(async move {
+        let provider = ProviderBuilder::new()
+            .on_ipc(IpcConnect::new(config.ipc_path.unwrap()))
```

Comment:
Is this change still necessary? We can fail with `?` and do not need to cancel `global_cancel`.

Reply:
Hi, using `?` here would mean that we would have to change the return type of the tokio::spawn to a `Result` instead of `()`, I looked at other spawned tasks (clean_job, rpc_server) and they all handle errors internally, so I thought I should do the same...
Also, for `global_cancel` - if the user specified a path to the IPC and it failed to connect, is shutting down not appropriate?
I'm happy to make both changes, just asking some questions for clarity

Reply:
That happens because you put the ProviderBuilder inside the spawn task. In the original code the provider is created outside. Would reverting to that work?
Also, remove the `unwrap` from the config.ipc_path.